### PR TITLE
Remove unneeded texture memory once loaded to GPU and Lightmap control in FBX

### PR DESCRIPTION
**Pull request by samcake**
- Notify the gpu::texture::storage once the mip pixels have been loaded in GPU memory so the STorage can free the memory. THis shoudl brings back to the memory usage of before my PR introducing gpu::texture end of january

- In the FBX file, in the "hifi_global" light, we capture the color and use the color.x as the offset parameter for the lightmaps of the model

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: d4566bb1d7cd848d1c9bc15bf09c8d62fb5c727f available at: https://gist.github.com/99f173f64967ea885913<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4248.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4248.dmg

File: libraries/gpu/src/gpu/Texture.cpp [hunk 1]
```diff
@@ -53,6 +54,14 @@ const Texture::PixelsPointer Texture::Storage::getMip(uint16 level) const {
     return PixelsPointer();
 }
 
+void Texture::Storage::notifyGPULoaded(uint16 level) const {
+    PixelsPointer mip = getMip(level);
+    if (mip) {
+        mip->_isGPULoaded = true;
+        mip->_sysmem.resize(0);
```
Comment (ZappoMan) [inline libraries/gpu/src/gpu/Texture.cpp]: don't you want to #ifndef WIN32 here??

Comment (ZappoMan) [inline libraries/gpu/src/gpu/Texture.cpp]: jk

Comment (samcake) [inline libraries/gpu/src/gpu/Texture.cpp]: :)

**Pull request merged**